datafusion/src/physical_plan/windows/window_agg_exec.rs

```diff
@@ -143,6 +143,18 @@ impl ExecutionPlan for WindowAggExec {
         ));
         Ok(stream)
     }
+
+    fn fmt_as(
+        &self,
+        t: DisplayFormatType,
+        f: &mut std::fmt::Formatter,
+    ) -> std::fmt::Result {
+        match t {
+            DisplayFormatType::Default => {
+                write!(f, "WindowAggExec")
```

Comment:
I think might be worth including self.window_expr (aka `    window_expr: Vec<Arc<dyn WindowExpr>>`) as well

	You can check out https://github.com/apache/arrow-datafusion/blob/master/datafusion/src/physical_plan/hash_aggregate.rs#L260-L272 for an example

Reply:
Should be resolved in https://github.com/apache/arrow-datafusion/pull/939/commits/c5e46b131535b21698860f1e2c5aaec40b53c4ce.diff